### PR TITLE
[WEB][BOTH][CRITICAL] Adding new test case TC001 and modifying DemoTest2

### PR DESCRIPTION
**Pull request by souravtiwary95**
**What is this PR for?**
 To add a new tet case TC001 and to modify DemoTest2

**Files Modified?**
/GitDemoProjectDemo1/src/test/java/com/qa/test/DemoTest2.java

**Library files added/modified?**
NA

**Files newly added?**
/GitDemoProjectDemo1/src/test/java/com/qa/test/TC001.java

**Jenkin Execution result?**
Jenkin success job

### CODE REVIEW

File: src/test/java/com/qa/test/TC001.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.qa.test;
+
+import org.testng.annotations.Test;
+
+public class TC001 {
+
+	@Test
+	public void m1()
+	{
+		System.out.println("This is a newlt added Test Case");
```
Comment (souravanand95) [inline src/test/java/com/qa/test/TC001.java]: some gramatical mistake is there, fix it.

File: src/test/java/com/qa/test/TC001.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.qa.test;
+
+import org.testng.annotations.Test;
+
+public class TC001 {
+
+	@Test
+	public void m1()
```
Comment (souravanand95) [inline src/test/java/com/qa/test/TC001.java]: make the method more meaningful

**Review by souravanand95 — COMMENTED** [review]
complete it today without fail

File: src/test/java/com/qa/test/TC001.java (hunk 1 above)
Comment (souravtiwary95) [reply src/test/java/com/qa/test/TC001.java]: ok I will do it

File: src/test/java/com/qa/test/TC001.java (hunk 2 above)
Comment (souravtiwary95) [reply src/test/java/com/qa/test/TC001.java]: finr

**Review by souravanand95 — COMMENTED** [review]
**The next build is about to come, so do this build before the new build comes. Thankyou**

**Comment by souravtiwary95** [comment]
Hi,
I have made the changes, kindly check it and approve.
Thankyou
Regards Test Engineer

**Review by souravanand95 — APPROVED** [review]
Code looks fine now, good job

**Pull request merged**